measure_cocoapod_size.py

```diff
@@ -27,6 +27,7 @@
 import tempfile
 from collections import OrderedDict
 from xcode_project_diff import GenerateSizeDifference
+from utils import shell
```

Comment:
I see in another file in the PR you removed a duplicate `#!/usr/bin/python`. There is another duplicate in this file on line  17.